src/Test.UnitTests.Sarif/FileRegionsCacheTests.cs

```diff
@@ -645,5 +645,21 @@
             multilineRegion.CharLength.Should().Be(content.Length);
             multilineRegion.Snippet.Text.Should().Be(content);
         }
+
+        // Placeholder for API Test
+        [Fact]
+        public void SuvamTest()
```

Comment:
>SuvamTest [](http://example.com/codeflow?start=20&length=9)

is this the name you want?

Reply:
@michaelcfanning removed. This was a placeholder. Thanks!